src/lib/draw/HoverLayer.svelte

```diff
@@ -13,14 +13,18 @@
   import {
     formOpen,
     gjScheme,
+    isAToolInUse,
     map,
     mapHover,
     sidebarHover,
   } from "../../stores";
 
   // Show clickable objects on the map using the cursor
   $: {
-    $map.getCanvas().style.cursor = $mapHover ? "pointer" : "grab";
+    // Don't override what an active tool has set
+    if (!$isAToolInUse) {
+      $map.getCanvas().style.cursor = $mapHover ? "pointer" : "inherit";
```

Comment:
`inherit` instead of `grab` to let maplibre set grab or grabbing as appropriate. Basically saying "we don't care"